server/services/calendarService.js

```diff
@@ -2,8 +2,8 @@ const axios = require('axios')
 const logger = require('../../log')
 const utilities = require('../helpers/utilities')
 
-module.exports = function CalendarService() {
-  function getCalendarData(apiUrl, startDate, accessToken) {
+module.exports = {
```

Comment:
I guess this is a stylisitic preference, I am using the [method shorthand syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions) as I see this the service as a collection of methods.

Can change over if you prefer.

Reply:
I wasn't questioning the validity of the approach, I think the shorthand is clearer with utility like short (<5 line) functions. These are over twenty lines each so at a glance, I struggle to see what the exports from this file are. Again personal preference. Not a show stopper 👍

Reply:
So this may be me being pedantic, but I often write and define my functions and methods differently.

(my definitions of each)
functions - individual functions that are intended to be used separately
methods - groups of related functions that have architectural relavance eg a service or data model

I'll often define functions as individual variables and export them as such
```
module.exports =  { getCalendarData,	getCalendarDetails }
```

And when I am writing a service or data model I will define them grouped together as a collection
```
module.exports = {
  getCalendarData(apiUrl, startDate, accessToken) {}
}

//or

const CalendarService = {
  getCalendarData(apiUrl, startDate, accessToken) {}
}

module.exports = CalendarService
```

I guess I am defining them in a certain way to imply intent of use. This is all stylisic of course, but I would prefer to leave as is as it helps me conceptualise how it all fits together.